client/editSchedule.js

```diff
@@ -1,37 +1,27 @@
-var editingRoundReact = new ReactiveVar(null);
+var eventToEditReact = new ReactiveVar(null);  // TODO I don't think this needs to be reactive - Lars
```

Comment:
What confused me is that it's not referenced directly. So... Blaze senses that `eventToEditReact` is touched during rendering of the dialog, and then rerenders it automatically whenever the variable is changed?

That does make sense. I'm not used to thinking this way yet.